tests/test_model.py

```diff
@@ -59,9 +66,13 @@ def test_set_debug(self):
 		self.assertRaises(model.MetadataError, model.factory.maybe_warn, "test")
 
 	def test_load_context(self):
-		self.assertRaises(model.ConfigurationError, model.factory.load_context, 
-			"foo", {"foo":"does_not_exist.txt"})
-		model.factory.load_context("foo", {"foo":"tests/test_context.json"})
+		self.assertRaises(model.ConfigurationError, model.factory.load_context, "foo", {"foo":"does_not_exist.txt"})
+
+		# gvp - win path fix ('curr_dir' defined earlier)
+		model.factory.load_context("foo", {"foo": curr_dir + "test_context.json"})
```

Comment:
This should use os.path.join() not +, as + does not append the correct path separator causing the tests to fail.